subprojects/testfx-core/src/main/java/org/testfx/api/FxAssert.java

```diff
@@ -38,6 +35,11 @@
 import org.testfx.service.finder.NodeFinder;
 import org.testfx.service.query.NodeQuery;
 
+/**
+ * All TestFX tests should use {@link #verifyThat(Node, Matcher, Function)} when writing tests,
```

Comment:
Why? I really meant all tests should use the `verifyThat(Node, Matcher)`-related assertions because if something goes wrong, one can quickly add an error handler (type now becomes `verifyThat(Node, Matcher, Function)`) that provides additional info to help one debug it. This applies to both TestFX contributors and users of this test library